src/Bundler/Pipeline/ContentPipeline.php

```diff
@@ -94,7 +94,7 @@ public function push(array $dependencies, ReaderInterface $file_reader, File $ta
             $module_name = $file->getName();
 
             if (!empty($this->config->getSourceRoot())
-                && false !== strpos($module_name, $this->config->getSourceRoot())
+                && 0 === strpos($module_name, $this->config->getSourceRoot())
```

Comment:
This is a fix for probably a bug that exists for which no unit-test has been added.